atom/renderer/api/lib/desktop-capturer.coffee

```diff
@@ -0,0 +1,10 @@
+ipc = require 'ipc'
+NativeImage = require 'native-image'
+
+getSources = (options, callback) ->
+  ipc.send 'ATOM_BROWSER_DESKTOP_CAPTURER_GET_SOURCES', options
+  ipc.once 'ATOM_REDNERER_DESKTOP_CAPTURER_RESULT', (sources) ->
```

Comment:
ATOM_REDNERER_DESKTOP_CAPTURER_RESULT is misspelled here, it should be ATOM_RENDERER_DESKTOP_CAPTURER_RESULT

I'm also seeing this file leaking listeners, in the console when calling `getSources`
```
(node) warning: possible EventEmitter memory leak detected. 11 destroyed listeners added. Use emitter.setMaxListeners() to increase limit.
```